### ISSUE TITLE
Can't save link field in matrix in Craft 3.4

### ISSUE THREAD

**Issue by ryanmwc** [sebastian-lenz/craft-linkfield#89]
I updated to Craft 3.4.0.2 from 3.3.x this morning and tried adding a linkfield to a matrix block, but I get this error in the web log:
```
yii\base\ErrorException:1] yii\base\ErrorException: Unknown: Input variables exceeded 1000. To increase the limit change max_input_vars in php.ini. in /Users/me/Sites/mySite/vendor/yiisoft/yii2/base/ErrorException.php:43
```
It only happens in a matrix field. I tested creating a linkfield as its own custom field and everything was fine.

**Comment by sebastian-lenz** [comment]
Hi @ryanmwc and thanks for reporting this. I've checked both v1 and v2 of this plugin in Craft 3.4.1 and the both seem to function well within matrix fields. Could you
- check whether the issue still exists with the latest version of Craft 3.4
- if so, give additional details on how to reproduce the issue

**Comment by ryanmwc** [comment]
@sebastian-lenz I've discovered that it happens even if just trying to add a new block type to this matrix field with a plain text field. It might be a Craft bug or it could be something with my setup after going through the 3.3.x to 3.4 update.  I posted an issue in Craft's issues: https://github.com/craftcms/cms/issues/5554

I'll trying testing with more variations and reply back when I get it sorted.
